### PR TITLE
Added seasonal categories.

### PR DESCRIPTION
**Pull request by LinoxGH**
Added Seasonal-Categories.md
Updated Dev guide.

### CODE REVIEW

File: pages/Developer-Guide-(3-Your-first-Item).md [hunk 1]
```diff
@@ -85,6 +85,25 @@ Category category = new Category(categoryId, categoryItem);
 The Category will not be visible in our Slimefun Guide at this point though.<br>
 We first need to add an actual SlimefunItem.
 
+### Seasonal and Locked categories
+You can also create a `SeasonalCategory` or a `LockedCategory` instead of a generic `Category`.<br>
+These types of categories require a specified category tier. This integer roughly determines the position
+of the category inside Slimefun guide. The guide starts populating with tier 1 and onwards. The other criteria
+is the order of registering(creation of `Category` object).<br>
```
Comment (svr333) [inline pages/Developer-Guide-(3-Your-first-Item).md:92]: a space between registering and the opening bracket would be nice

File: pages/Seasonal-Categories.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Seasonal categories, are categories that can only be accessed in certain months of a year.
```
Comment (svr333) [inline pages/Seasonal-Categories.md:1]: `during a certain month every year` feels better

**Review by svr333 — COMMENTED** [review]
These changes arent needed but seem beneficial

**Review by svr333 — APPROVED** [review]
Looks good to me

**Review by Sfiguz7 — APPROVED** [review]
Seems fine to me too

**Pull request merged**